.github/workflows/pr-preview.yml

```diff
@@ -0,0 +1,28 @@
+name: Deploy PR previews
+
+on:
+  pull_request:
+    types:
+      - opened
+      - reopened
+      - synchronize
+      - closed
+
+concurrency: preview-${{ github.ref }}
+
+jobs:
+  deploy-preview:
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Install and Build
```

Comment:
Q: do we need to load node?

Reply:
i think, yes... because we need to build the dependencies to create a build `dist` for `prepare:ghpages`